### PR TITLE
 Create an entity for scheduled activities (scheduling)

### LINKED ISSUE #17: Create an entity for scheduled activities (scheduling)
Create an entity for scheduled activities #5, where the information from the completed scheduling session is stored.

### PR DESCRIPTION
**Pull request by SamuelBruegger**
closes #17 

- added ScheduledActivity class

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs21/entities/ScheduledActivity.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        this.id = id;
+    }
+
+    public UserActivity getChosenActivity() {
+        return chosenActivity;
+    }
+
+    public void setChosenActivity(UserActivity chosenLocation) {
+        this.chosenActivity = chosenActivity;
+    }
```
Comment (MESC2020) [inline src/main/java/ch/uzh/ifi/hase/soprafs21/entities/ScheduledActivity.java]: the parameter should be "**chosenActivity**" and not chosenLocation

**Comment by MESC2020** [comment]
I'm not sure how this works with entities in Spring Boot context but don't you have to define a constructor?

File: src/main/java/ch/uzh/ifi/hase/soprafs21/entities/ScheduledActivity.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    private Long id;
+
+    @Column
+    private UserActivity chosenActivity;
+
+    @Column
+    private String chosenLocation;
+
+    @Column
+    private Date chosenDate;
```
Comment (Taremeh) [inline src/main/java/ch/uzh/ifi/hase/soprafs21/entities/ScheduledActivity.java]:
```suggestion
    private LocalDateTime chosenDate;
```
probably of type `LocalDateTime` containing date & time? If so, getter & setter need to be updated as well

**Review by Taremeh — CHANGES_REQUESTED** [review]
To simplify the naming we could omit the `chosen` in all variables and method names since it's clear that e.g. the location of a ScheduledActivity is the location that has been chosen. Or is there any benefit of keeping it?

**Review by MESC2020 — APPROVED** [review]

**Pull request merged**